runtime/integration-tests/src/liquidity_pools/pallet/development/tests/liquidity_pools/transfers.rs

```diff
@@ -73,13 +73,17 @@ fn transfer_non_tranche_tokens_from_local() {
 		// Register GLMR and fund BOB
 		setup_pre_requirements();
 
-		let initial_balance = 100_000_000;
+		let initial_balance = 2 * AUSD_ED;
 		let amount = initial_balance / 2;
 		let dest_address = DEFAULT_DOMAIN_ADDRESS_MOONBEAM;
 		let currency_id = AUSD_CURRENCY_ID;
-		let source_account = BOB;
+		let source_account = CHARLIE;
```

Comment:
Not sure why but apparently Bob was funded with the ED.

Reply:
Yes, by default, I added `ED` to Alice and Bob. We need to discuss how to initialize tests to avoid this kind of unexpected initialized state (I'm working on some experimental currency management in the generic integration tests to clarify this aspect).

Reply:
How about no default funding but instead builder functions for the test environment which enable funding a vector of account addresses? 

Example
https://github.com/centrifuge/centrifuge-chain/blob/90d3cf9f2884489dafbc221925b2608ced4ace5e/pallets/block-rewards/src/mock.rs#L327

Reply:
I'll think about it!